src/invisible/how-tos/advanced-analysis/intrinsics/README.ipynb

```diff
@@ -0,0 +1,336 @@
sQ64XptMG4/1B+/Rxp7sqcn4SkmzKR7gCEKXKmoByVoLFpysvyI2Eoy4VbPtQXONbTUVLODJKrWDdneYXZHA+F8PuPp6QnzPGF32GkoiKFNQG0MU0pqTaYEEe0ePY4Bu/0EgCCFcDweQSDs9wcAmgg/DAPuDvcYpgFFClJSb4Z+fwIgOL0csSzJPBsantrNM4Zh0IRwEewOOwyDWrPZyq3V9hE4V4uCVIOIBFuzqKWn6vuw6iSfSfuf7/06ch3ale6zXibPgY3sT3PBLpeLJqPHqLxXoRlLrmDcY+Mypq4O6YBGbzCKNk2VSh4qFsZiU47KzW3tMmsYchtag1+4WxH9QUANtznERvfTFNV/UcctqNaeWrbDv9kv9eN/8TX7dxoBIrMghIIgSnipXr2EeZ6x3+3w8cMHnF6O+PPPf8Yvn7/gcllqtZPmS1m/ulIsyZ4snUbqg5EbciZD1iTqtTdjGgCENTHYm8965aSvBXKOJSJIKVjXgrQsKOVsERYF4zV/iwqo+88J8JQnV2p/pppPRJ2XizQnLISAeVIDJpjHGVSQ0l9Qfv1eHgAHRthPwDRAQtJGhkVLaVlUUKo3wwFLUbpjUQZAsjJnUjYgUMkIxfIlbFOXjsjNQ1KAlxKre32w7OdgrKRipHapZCW9MyFY3JtgQIZs75LRsDfkqApC51DMMqM6HsoqmnC5nG2i1ZMkSZ9fxLgOjQadN8rFOSz0+swBP/30E/7uf/ofEQ87pDFCiHG43+Phhw94ev6KtJzAWUeCsuaapNMFl6/PONx/xBSB+92EU5gAucATpiIx8nnBr//wH/HLP/4JIY6IZJ2DQdjb5MdlxSFEkAjWos+WlhVhTQCpu1AXq5j1XraOOcCAhyCzIEnGZdX+TFI6IGMLeRgsudeSvlwRO6vqxpOSAYqavH05J6zLc+2N01flAAUhkHVi91AorHWFJWLeOKrwqqB1ayVfl9C2633n4R49YBOu1G0tm5wO7Q0WDMSoVVIyI2TdL7AWBiC2xL5mJFx7ofwu6zSJC1Dz1oAwDLECmXEccX9/B+KINQvWlBHjaMDP16qFVYaCdVmwLBc8PT1iWVZACLlk7HYKNOZxwv19wOFwBw/HjNMEQcEvv/6Cf/rTn7CcLxAI9rs7/PjDTxjGgP3+UMfhYCXX3sNJ5hkxRKyr8tQsOWs4EkHL9710GX0OTDNcKt8Tmk4ndTeZceUhZc0tEDOgiBwAOxi0GSMDjpa7lRIsf+aCoWTkoqCMfZ37evI8s25RbWese70uR+1TpuEDAUishQeZOCMIWUKkh9QczJC77OVqHTfF7lWM1x6i/sd/WcctENN+3vCd/M1vx/d5vRoFBAgYAcwBUiIkF4xxxDzOuNvf4XC4xw+fvuLp+Rm/fv6M55cXXJYF2ZLNPSCZC6x1EVfelxAZHHRKC4wXSVbzWhO8i7aGhBQsSLY1IJr7WYp2uyeTV86taTtGu19D2zAAonmR1QbwqkxBe2jUMHefr+erEKScUdM0YYgDBjYqFwFqz8I3jm8CmRZn7eLxUArhad7jMu8AOuuTgyCpIFOCSEBZE0Q03CFG95+hpbWtkaM9hHRslnC3kwKTIu6R0NCDCh9XhsqhomWRUdleibBeFpwvi1E+A0Te3Ra1g3Zgzb1gIus6rLk79oD63Cwwp4+FmFSAckew5V6JrXJtbzE3oj8Gm3XleT2CtK5avSSaFDaNAfcf7nCcByxPSqQ1QUEfJ0FYFqQvn3GZJkgYMKWMmQirWAKshXy4iFppLIg5Y44Bo+gzU8kYiTDHERMYEgQnSThK1sTZ0FzKAiBnTwJr5G1SSL0IlvukgAS4yKr5NNDNQBwsS90UeM3fUGCaS1LeAQMyzAzNeNAmmgLl82CWOqYFouMohtoro6sOv2fMV8xv1jo8Z0JQgSrsdV/zQAOvG5fNG8c3xWC18m1EbY6YyfaC0gawgZgwDIjjBEhjzWZoc07qgExjq26emZoLI9C5h4ZaQtDGd+MwYBoG3N/vlZMoaKfq3W7CPE2gEHBZi7KLBgAlIbB+fxwHiJEUnk4nPD4+4suXZ5xPi4KPuz3u7w748YePmOexAhFAw1tPT084vhwhWTBEbQY3jSOWyxkx7HB3uAOTgqxpmiqYAvRZf/38K06nE9Y1YZh2oKDNJb2lQIvkuGJu87g1Rtpe5Qp+fH4YEKqGjFQAcA0EUM/HgTEFvb7yw+ieFhEl8hsHJdv03KcKmnrPj9/ndlmyuVPIV4IIkK3qk6MiUaZaWuvI3MMPta/wDYzy39xxjdW61wH8TfHMZsirsaFqPFi4Hqx5UN4CJ9wFTJMaGT98+ojnF610+vL1EcfTCUnUYIzB4hMWetY6nIAQDay4IarN75Cz6lEHQlUUmt5QbKwJwFQKyL3DgeFkncSMIJoeomC9hbU3uVlEBl7UGKLAm9C3+vpant80TZjmSZvFsofAvz0x3wgtZdsKr92KzIwYAniKWAagHDOmYn1hSkG5FO3BIgWEgmCVRDV5tbrAuslmS1qClkvCXVFZs6E1GlVQJKmS8zBEIOzmEaPllVzWFS8vR6QlmdtZrOGjVlRpJVHQrteBEQPUFbcRaPaTSb0sJFopJYLBwiOATYgIEM2M8wXRLdpqdQtr0ph15k3rBf/8p3/E83rB/2O/w/hjBFFBZGA/q4t9YfNKFS3DjgxMYPD5jMs//Qm5ENbjgng8qfC08mU4ey3UQxYkIZaAOUSMwft0ZAQUSNEck1QyUtKU2WLhL4LmPWh1FrSKKIs24CRLQpMWNispK5qnFUxKdLbbDXD0rxn1gilGXEJAyQkQy2FofnEFhlWiexK2oXKS6vUW56WpSWbNQ9FgGFDFhnsv6it2ns6b8VuOrQ19/WZ7p7Fx2vOECC8iYIImNceoneSnHUIcwNAwSiwFGUp46BYTkSfWasm830hLWAfGcWdVExqDPxzulIMlRPPIuLfCPZSa0AowkgjY3ACn4xnrekGM2tYAVkY/T
```

Comment:
@dourvaris I copied this code from an example you created on how to copy an audio stream over properly. When adding line 238 I get an error though:
`ValueError: Can only set extradata for decoders.`

Seems to run fine without this line though. Can you confirm that we can just drop it?

Reply:
Yes